SPECS/kernel/kernel.spec

```diff
@@ -404,6 +404,9 @@ echo "initrd of kernel %{uname_r} removed" >&2
 %{_sysconfdir}/bash_completion.d/bpftool
 
 %changelog
+* Wed Mar 27 2024 Cameron Baird <cameronbaird@microsoft.com> - 6.6.22.1-2
+- Bring aarch64 kernel config into alignment with x86 for hv, xen, virtio drivers
```

Comment:
Reading this changelog entry, the entry states that you are bringing the aarch64 kernel config build in alignment with x86_64 kernel config, but there is no fundamental reason why the aarch64 kernel config for these drivers should be in alignment with x86_64. From an outside view, it looks like this change is simply because you saw a diff between x86_64 and ARM64 configs and wanted to clean it up.

But there is a real reason behind why you are making this change.

Can you put that reasoning here instead, and drop the x86_64 reference, because you are actually just changing the aarch64 config to produce hv, xen, virtio as modules in this kernel package change in order to support dracut initramfs generation on arm64 VM systems.